app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -247,7 +247,7 @@ class MapFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickListe
                 }
             } else if (resultCode == AutocompleteActivity.RESULT_ERROR) {
                 // TODO: Handle the error.
-                var status = data?.let { Autocomplete.getStatusFromIntent(it) }
+                //var status = data?.let { Autocomplete.getStatusFromIntent(it) }
```

Comment:
why are we commenting out this code? can we remove it?

Reply:
I didn't remove it because it said TODO: 

Correct me if I am wrong, I am assuming we will implement this in the future using that line of code?

Reply:
This entire method is actually useless now. We have no use for onActivityResult() at the current moment. 
It can safely be discarded.

Reply:
It is mentioned in comments that we should check the result of a location settings check failure in onActivityResult() but this wasn't curently implemented in the method to begin with. Further map fragment refactoring should look into that.

Reply:
Update: Digging through commit history I found that the callback was implemented in the past but somehow got reverted. I will change it to what it was for now. The current code with Autocomplete is definitely not needed anymore though.